### ISSUE TITLE
Wrong Creation Date 

### ISSUE THREAD

**Issue by flribeiro1983** [MartinSGill/TodoTxtJs#83]
When I create a task, the "creation date" appears as "Yesterday", when it should be "Today". Is this a bug?

**Comment by flribeiro1983** [comment]
Here's an image showing a task I just added, with "Yesterday" creation date. Using Google Chrome in Windows 10.

![todotxtjs](https://cloud.githubusercontent.com/assets/14063070/9594245/f614a256-502e-11e5-83c8-3ee57c662184.PNG)

**Comment by MartinSGill** [comment]
I have been unable to reproduce this. I always see it as "today". Is there any additional information you can provide me? e.g. does it only happen near midnight?

**Comment by flribeiro1983** [comment]
The problem occurred during all day, anytime. I was using Google Chrome on
Windows. The same occurred at my home PC or at my work PC.

Cordialmente,

*FABRÍCIO LOMBARDI RIBEIRO*
Tel. +55 16 98185-2222 | Skype: fabricio.tob.brasil
LinkedIn: http://br.linkedin.com/in/flribeiro
Twitter: @flribeiro <http://twitter.com/flribeiro>

**Comment by MartinSGill** [comment]
I wonder if it's time zone related, there have been bugs around that before. I presume the actual date is correct and the issue is only a cosmetic one?

**Comment by flribeiro1983** [comment]
Exactly! I was using the same todo.txt file in another app simultaneously
and that it was exhibited correctly.

Cordialmente,

*FABRÍCIO LOMBARDI RIBEIRO*
Tel. +55 16 98185-2222 | Skype: fabricio.tob.brasil
LinkedIn: http://br.linkedin.com/in/flribeiro
Twitter: @flribeiro <http://twitter.com/flribeiro>

**Comment by leofontenelle** [comment]
Although I didn't report it yet, I have the same problem. I have the same timezone (Americas/Sao_Paulo) and language (pt_BR) as Fabrício Ribeiro. The issue happens with Chrome on Windows or Android as well as with Firefox (and IIRC other browsers) on Linux.

Although the todo.txt file itself has the correct date, TodoTxtJs (at least in MartinSGill's server) displays "Yesterday" (in English) as the human friendly date.

Is there any test you'd need us to to in order to understand what causes this?

**Comment by cliftonprince** [comment]
I have the same problem. Before I found this thread on the bug, I wrote up the following report to submit as a "New Issue," but then I found this thread so I didn't submit it that way. I might as well include the text of it, but it's just describing the same existing problem. 

> 
> Is this you, time-zones, web-server, something else, or just me? Dates are off by a day. 
> 
> Today is Monday, February 1, 2016. It is 11 a.m. in the Central Time Zone of the United States. I have a todo.txt file with tasks in it that are dated as created today with the proper text-string "2016-02-01". But the web interface at http://todo.martinsgill.co.uk/? claims that they were created "Yesterday". Similarly, I set up a new task in the web interface with a creation-date of "2016-02-08", which is also a Monday. The web interface indicates it will be created on "Sunday". Further, I set up a new task in the web interface with a creation-date of "2016-01-31", but it changed the creation-date to "2016-01-30".
> 
> Thought you'd like to know. No skin offa my back. :)
> 
> cliftonprince>>>@>>>cliftonprince.com no need to reply best wishes

**Comment by emanderson** [comment]
I am having the same problem, using Chrome on my Mac in the Pacific time zone. The creation dates in the text file itself seem to be fine, but the dates in the formatted view are a day earlier. So far not a deal breaker for me but a bit confusing.
